.github/workflows/firesim-cleanup.yml

```diff
@@ -32,6 +32,14 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/repo-setup-aws
       - run: .github/scripts/cull-old-ci-instances.py
+      - if: ${{ failure() }}
+        uses: rtCamp/action-slack-notify@v2
+        env:
+            SLACK_CHANNEL: firesim-github-update
+            SLACK_COLOR: ${{ job.status }}
+            SLACK_TITLE: "CULLED CI INSTANCES (PLEASE VERIFY CI IS WORKING CORRECTLY!)"
```

Comment:
Since this cron job isn't tied to a particular PR, just email the Slack channel directly.

Reply:
Which channel is this updating?